examples/debugging/scripts/hybrid_debugging.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2018 Owkin, inc.
```

Comment:
could be renamed without the word "hybrid"?

Reply:
Indeed, is `debugging.py` OK with you ? In the sense that we have a 'debug' mode and a specific use of this mode is the 'fully local' debugging.

Reply:
Perfect, thanks!